PocketKit/Tests/PocketKitTests/AuthorizationClientTests.swift

```diff
@@ -57,7 +62,7 @@ extension AuthorizationClientTests {
         }
     }
 
-    func test_logIn_startsOnlyOneSession() throws {
+    func test_logIn_startsOnlyOneSession() async {
```

Comment:
removing Task and using `async throws` cause this test to hang. not sure why though